tests/mail/tests.py

```diff
@@ -305,6 +307,158 @@ def test_non_ascii_attachment_filename(self):
         payload = message.get_payload()
         self.assertEqual(payload[1].get_filename(), 'une pièce jointe.pdf')
 
+    def test_attach_file_text(self):
+        email = EmailMessage('subject', 'body', 'from@example.com', ['to@example.com'])
+        basename = 'file.txt'
+        self.assertEqual(mimetypes.guess_type(basename)[0], 'text/plain')
+        self.assertEqual(email.attachments, [])
+        email.attach_file(os.path.join(os.path.dirname(upath(__file__)), 'attachments', basename),
```

Comment:
seems like a helper method to get the attachment path would save some repetition